### ISSUE TITLE
minLength 0 potential bug

### ISSUE THREAD

**Issue by schang933** [sergiodlopes/jquery-flexdatalist#14]
I could reproduce this with the following code on Chrome and Safari on OS X.

* Click on input
* Select any element
* Click on input
* Ctrl-A || Cmd-A || (right click && select all)
* Press backspace
* The selected element has disappeared

To make it reappear, you need to blur and refocus. Note that I could not reproduce it on your demo page, so it is possible that there is something wrong with the code below. Please let me know if that's the case.

```html
<link rel="stylesheet" href="jquery.flexdatalist.min.css">
<script src="https://code.jquery.com/jquery-3.1.0.min.js" integrity="sha256-cCueBR6CsyA4/9szpPfrX3s49M9vUU5BgtiJj06wt/s=" crossorigin="anonymous"></script>
<script src="jquery.flexdatalist.min.js"></script>

<style type="text/css">
  input {
    width: 100%;
  }
</style>

<input class="yesNoData" type="text" name="dataField">

<script type="text/javascript">
$(document).ready(function() {
  $('.yesNoData').flexdatalist({
    minLength: 0,
    valueProperty: 'value',
    visibleProperties: ['name'],
    searchIn: 'name',
    data: [
      {
        name: 'Yes',
        value: 'YES'
      },
      {
        name: 'No',
        value: 'NO'
      }
    ]
  });
});
</script>
```

**Comment by sergiodlopes** [comment]
Hey,
Please check if is fixed on 1.7.0! Thanks.
Sérgio

**Comment by schang933** [comment]
Awesome, went through the closed bugs and looks great. Thanks!
